packages/fast-components-react-base/src/divider/divider.props.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+
+/**
+ * Divider HTML Roles
+ */
+export enum DividerRole {
+    presentation = 'presentation',
+    separator = 'separator'
+}
+
+export interface IDividerProps {
```

Comment:
I proposed a new naming convention for props interfaces in my snapshot PR - this interface is representative of "handled" props so I think it should be named accordingly. Take a look at https://github.com/Microsoft/fast-dna/pull/207 in the button.props.ts file for details.

Reply:
this makes sense for better readability.